Automattic/sensei
# Course List UI improvements for Astra

### Changes proposed in this Pull Request
This PR addresses a couple of UI issues in Astra, namely:
- Applying a margin between the featured image and the categories in the editor
- Ensuring categories aren't underlined

### Testing instructions

- Switch to the Astra theme.
- Ensure there is enough whitespace between the featured image and the rest of the course details.
- Ensure categories aren't underlined.
